test/test_torch.py

```diff
@@ -18995,7 +19003,11 @@ def test_nansum_out_dtype(self, device):
             torch_fn = partial(torch.nansum, dtype=out_dtype)
             np_out_dtype = torch_to_numpy_dtype_dict[out_dtype]
             np_fn = partial(np.nansum, dtype=np_out_dtype)
-            self.compare_with_numpy(torch_fn, np_fn, x, device=None, dtype=None)
+            if (inp_dtype, out_dtype) == (torch.uint8, torch.float16):
+                # 25504.0 vs 25536.0
+                self.compare_with_numpy(torch_fn, np_fn, x, device=None, dtype=None, atol=50, rtol=0)
```

Comment:
can you make it rtol=0.002 or something, and less outrageous atol?

Reply:
fixed